chargebee-js-vue/src/components/Element.vue

```diff
@@ -47,13 +47,14 @@ export default {
       if (cbComponent) {
         const options = this.fieldOptions;
         this.field = cbComponent.createField(this.id, options).at(`#${this.elementId}`)
-        if (this.$parent.onMount) this.$parent.onMount()
+        if (this.$parent.onMount) this.$parent.onMount();
 
         // Attach listeners if any
         this.attachListener('ready')
         this.attachListener('focus')
         this.attachListener('blur')
         this.attachListener('change')
+        this.initialized = true;
```

Comment:
`initialized` variable to be added to the `data` attributes

Reply:
Done.